### DISCUSSION TITLE
New to Mastodon - Translate feature

### DISCUSSION THREAD

**Discussion by jpowzy** [mastodon/mastodon#19008] [General]
Hi! New to Mastodon. One feature I'd like to see implemented is a translate feature. Its been decades since I wrote a line of code so its beyond the scope of my abilities but would be a handy feature to have implemented.

**Comment by shleeable** [comment]
There has been a lot of discussion on this topic.

https://libretranslate.com/ or similar coming soon

**Comment by yelvington** [comment]
https://github.com/mastodon/mastodon/discussions/20903
